dotnet-authserver/src/TeacherIdentity.AuthServer/Pages/Account/Name/Confirm.cshtml

```diff
@@ -0,0 +1,20 @@
+@page "/account/name/confirm"
+@model TeacherIdentity.AuthServer.Pages.Account.Name.Confirm
+@{
+    ViewBag.Title = "Confirm change";
+
+}
+
+@section BeforeContent
+{
+    <govuk-back-link href="@LinkGenerator.AccountName(Model.ReturnPath)" />
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds-from-desktop">
+        <form action="@LinkGenerator.AccountNameConfirm(Model.FirstName!, Model.LastName!, Model.ReturnPath)" method="post" asp-antiforgery="true">
```

Comment:
Think there's a summary list missing here: https://get-an-identity-prototype.herokuapp.com/account/check-answers

Reply:
It didn't look like it from the prototype https://get-an-identity-prototype.herokuapp.com/account/email